experimental/devices/ccs811/ccs811.go

```diff
@@ -0,0 +1,361 @@
+package ccs811
+
+import (
+	"fmt"
+	"time"
+
+	"periph.io/x/periph/conn"
+	"periph.io/x/periph/conn/i2c"
+)
+
+// Different measurement modes constants
+const (
+	MeasurementModeIdle         byte = 0
+	MeasurementModeConstant1000 byte = 1
+	MeasurementModePulse        byte = 2
+	MeasurementModeLowPower     byte = 3
+	MeasurementModeConstant250  byte = 4
+)
+
+// NeededData represents set of data read from the sensor
+type NeededData byte
+
+// What data should be read from the sensor
+const (
+	ReadCO2          NeededData = 2
+	ReadCO2VOC       NeededData = 4
+	ReadCO2VOCStatus NeededData = 5
+	ReadAll          NeededData = 8
```

Comment:
Frankly, I'd always read all and skip this flag altogether. It's not worth bothering the user with this, and the difference of 2 bytes vs 8 bytes read after the setup time, the address and the command being written is fairly low.

Not only this simplifies the driver, but it also simplifies the life of the user.

Reply:
I'm keeping both, sense is ReadAll, sensePartial is old version where user for some reason can read just something